kbase-extension/static/kbase/js/common/cellComponents/tabs/results/resultsTab.js

```diff
@@ -163,13 +163,13 @@ define(['bluebird', 'common/ui', 'common/events', './outputWidget', './reportWid
 
             const objectNode = document.createElement('div');
             const reportNode = document.createElement('div');
+            const containerNode = document.createElement('div');
+            containerNode.classList.add('hidden', 'kb-result-tab__container');
```

Comment:
Rejig slightly to add a container around the tab contents. This simplifies the styling slightly.